src/capitalsFirst.js

```diff
@@ -21,7 +21,25 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  if (!str) {
+    return 'String is empty';
```

Comment:
This message is not helpful. Based on the logic, if string is empty - same string should be returned.